MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: Manually create view items in .NET MAUI DataForm control | Syncfusion
+description: Learn about the working with manually created DataFormViewItems in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with manually created DataForm Items
```

Comment:
## Manually create Data Form Items

Data form autogenerate the editors based on the data type, you can explicitly add the data editors  by adding `SfDataForm.Items` manually and you need to change `AutoGenerateItems` property to false. 

Note
Use the FieldName property to bind the editor to the data object property. 



Add reference of table of DataFormItems available

Reply:
added